bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -0,0 +1,284 @@
+            <a class="mzp-c-cta-link" href="https://getpocket.com{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Pocket">
+              <img alt="" src="{{ static('img/logos/pocket/logo.svg') }}" height="40" width="40"><br>
+              {{ product_pocket }}
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="mzp-l-content">
+      {% call feature_card(
+        title='Firefox Browser',
+        heading_level=3,
+        image_url='firefox/home/master/browser.png',
+        include_highres_image=True,
+        class='mzp-l-card-feature-left-half t-browsers',
+        ga_title='Trackers',
+        media_after=True
+      ) %}
+        <h4>{{ browsers_trackers }}</h4>
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h5 class="mzp-c-menu-list-title">{{ get_browser }}</h5>
+          <ul class="mzp-c-menu-list-list" id="menu-browsers">
+            <li class="mzp-c-menu-list-item"><a href="{{ url('firefox.download.thanks') }}">{{ get_desktop }}</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK_UTMS }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Browser for Android"> {{ get_android }}</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ firefox_ios_url('mozorg-firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Browser for iOS">{{ get_ios }}</a></li>
+          </ul>
+        </div>
+        <p class="js-fx-only"><a class="mzp-c-cta-link" href="https://addons.mozilla.org/firefox/addon/facebook-container/{{ referrals }}">{{ get_facebook_container }}</a></p>
+      {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+        {% call feature_card(
+          title='Firefox Monitor',
+          heading_level=3,
+          image_url='firefox/home/master/monitor.svg',
+          class='mzp-l-card-feature-right-half t-monitor',
+          link_url='https://monitor.firefox.com/' + referrals,
+          link_cta=monitor_cta,
+          ga_title='Monitor',
+          media_after=True
+        ) %}
+        <h4>{{ monitor_title }}</h4>
+        {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+      {% call feature_card(
+        title='Firefox Lockwise',
+        heading_level=3,
+        image_url='firefox/home/master/lockwise.svg',
+        class='mzp-l-card-feature-left-half t-lockwise',
+        ga_title='Lockwise',
+        media_after=True
+      ) %}
+      <h4>{{ lockwise_title }}</h4>
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h5 class="mzp-c-menu-list-title">{{ get_app }}</h5>
+          <ul class="mzp-c-menu-list-list" id="menu-lockwise">
+            <li class="mzp-c-menu-list-item"><a href="{{ lockwise_adjust_url('android', 'firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Lockwise for Android"> {{ get_android }}</a></li>
```

Comment:
There's no Lockwise CTA here for desktop users - is that something we need to think abnout?

Reply:
There's a "Learn more about Lockwise" link beneath the menu. I think that's good for V1